<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Axis2DPropertiesDialog</class>
 <widget class="QDialog" name="Axis2DPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="scale">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeWidget" name="axisTreeWidget">
          <column>
           <property name="text">
            <string>Axes</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="axisFromLabel">
                  <property name="text">
                   <string>From</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="axisFromDoubleSpinBox">
                  <property name="minimum">
                   <double>-1000000000000000057857959942726969827393378689175040438172647424.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000000000000057857959942726969827393378689175040438172647424.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="axisToLabel">
                  <property name="text">
                   <string>To</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="axisToDoubleSpinBox">
                  <property name="minimum">
                   <double>-1000000000000000057857959942726969827393378689175040438172647424.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000000000000057857959942726969827393378689175040438172647424.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="axisTypeLabel">
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="axisTypeComboBox">
                  <item>
                   <property name="text">
                    <string>Linear</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Logarithemic</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="axisInvertedCheckBox">
                <property name="text">
                 <string>inverted</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="majorTicksLabel">
                  <property name="text">
                   <string>Major ticks</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="majorTicksSpinBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="minorTicksLabel">
                  <property name="text">
                   <string>Minor ticks</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="minorTicksSpinBox"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
